### PR TITLE
feat: Display list of contributors on project details page gf-337

### PR DESCRIPTION
**Pull request by GvoFor**
## Features
- add a list of contributor cards on the project details page
- there is only contributor name displayed in the card

## Screenshots
Contributors
image

"ToDo Application"
image

"ToDo Application 2"
image

"New Project"
image

### CODE REVIEW

**Comment by GvoFor** [comment]
P.s. I will resolve conflicts after #334 being merged

**Comment by what1s1ove** [comment]
@GvoFor please do it now

File: apps/backend/src/modules/contributors/contributor.service.ts
Comment (Vitaliistf) [reply apps/backend/src/modules/contributors/contributor.service.ts:81]: I'm not so sure about it, but looks like it would be more convenient to do this in repository, retrieve only needed data right away.

Comment (liza-veis) [reply apps/backend/src/modules/contributors/contributor.service.ts:81]: agree

File: apps/frontend/src/modules/contributors/slices/contributor.slice.ts
Comment (liza-veis) [inline apps/frontend/src/modules/contributors/slices/contributor.slice.ts:18]: This should be inside project slice and related actions too

**Review by liza-veis — CHANGES_REQUESTED** [review]

File: apps/frontend/src/modules/contributors/slices/contributor.slice.ts
Comment (GvoFor) [reply apps/frontend/src/modules/contributors/slices/contributor.slice.ts:18]: Good point

**Review by what1s1ove — APPROVED** [review]
lgtm after Lisa's comments

**Review by liza-veis — APPROVED** [review]

**Pull request merged**
